content/en/tutorials/schema-evolution-glue-msk/schema-evolution-glue-msk-flow.svg

```diff
@@ -0,0 +1 @@
low-wrap: normal;">Serializer</div></div></div></foreignObject><text x="70" y="83" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="14px" text-anchor="middle">Serializer</text></switch></g><path d="M 660 126.37 L 660 210 L 424.37 210" fill="none" stroke="rgb(0, 0, 0)" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 660 121.12 L 663.5 128.12 L 660 126.37 L 656.5 128.12 Z" fill="rgb(0, 0, 0)" stroke="rgb(0, 0, 0)" stroke-miterlimit="10" pointer-events="all"/><path d="M 419.12 210 L 426.12 206.5 L 424.37 210 L 426.12 213.5 Z" fill="rgb(0, 0, 0)" stroke="rgb(0, 0, 0)" stroke-miterlimit="10" pointer-events="all"/><rect x="590" y="0" width="140" height="120" rx="18" ry="18" fill="rgb(255, 255, 255)" stroke="rgb(0, 0, 0)" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe center; width: 138px; height: 1px; padding-top: 7px; margin-left: 591px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;"><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;"><b>Consumer</b></div></div></div></foreignObject><text x="660" y="21" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="14px" text-anchor="middle">Consumer</text></switch></g><rect x="600" y="49" width="120" height="60" rx="9" ry="9" fill="rgb(255, 255, 255)" stroke="rgb(0, 0, 0)" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 79px; margin-left: 601px;"><div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;"><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">Deserializer</div></div></div></foreignObject><text x="660" y="83" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="14px" text-anchor="middle">Deserializer</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
I think it would be nice to also migrate the [drawio file](https://github.com/localstack/localstack.github.io/blob/6926ca9197e3f474b4961d55352d30b878b17316/content/blog/2022-07-26-schema-evolution-glue-msk/schema-evolution-glue-msk-flow.drawio) from https://github.com/localstack/localstack.github.io/pull/57 to allow any future changes to the SVG without having to redraw everything from scratch.